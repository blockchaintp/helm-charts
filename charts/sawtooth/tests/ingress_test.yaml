suite: sawtooth.test the ingresses
templates:
  - ingress.yaml
release:
  name: test
  namespace: testns
tests:
  - it: ingress links to service properly
    set:
      ingress:
        enabled: true
        hostname: test-host
        path: /path
    asserts:
      - equal:
          path: spec.rules[0].http.paths[0].backend.service.name
          value: test-sawtooth
      - equal:
          path: spec.rules[0].http.paths[0].backend.service.port.number
          value: 8008
      - matchSnapshot:
          path: spec
