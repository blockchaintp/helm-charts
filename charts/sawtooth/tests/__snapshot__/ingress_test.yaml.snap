ingress links to service properly:
  1: |
    rules:
    - host: test-host
      http:
        paths:
        - backend:
            service:
              name: test-sawtooth
              port:
                number: 8008
          path: /path
          pathType: ImplementationSpecific
